src/sas/qtgui/Plotting/Slicers/ArcInteractor.py

```diff
@@ -73,27 +66,23 @@ def update(self, theta=None, phi=None, r=None):
         :param phi: angle from the centre point on the arc to each of its edges
```

Comment:
Why is it called "theta" for the angular value of the middle of the arc, whereas in the GUI its "phi", and here "phi" being the parameter that is called "delta_phi" in the GUI? That is very misleading.

Reply:
Here I was copying how the variables are named in SectorSlicer, which has exactly the same naming scheme. I agree it's confusing, and I'd be happy to change it for both of them.